### PR TITLE
Schedule draft

### PR DESCRIPTION
**Pull request by yochannah**
some of the mentor/mentee bits are in need of more detail but it seems like a good start :)

### CODE REVIEW

**Comment by malvikasharan** [comment]
Do you think it will be better to move schedule to a different page altogether? or move it to the about page?

**Comment by yochannah** [comment]
I think it's nice to have a bit of an overview on the main page, but I can remove it.

**Comment by yochannah** [comment]
@malvikasharan @bebatut I'll add some notes in #16 regarding which weeks I think each item from #16 should fit in to - plus please comment if you think we need to make changes :)

File: index.md [hunk 1]
```diff
@@ -39,21 +39,14 @@ Here are a few values we think you will receive from participating in this proje
 
 ## Schedule
 
-The first phase of Open Life Science will launch in early 2020!
```
Comment (bebatut) [inline index.md]: I would keep the important date here on the index page (that they got the information directly) and repeat them in the schedule page later

Comment (malvikasharan) [reply index.md]: Agree. The date will also keep changing (if we continue).

**Comment by bebatut** [comment]
I renamed the new page `syllabus` and started to fill content similarly to the MOL syllabus. I formatted the schedule as a table. 
Let me know if these changes make sense, otherwise I will revert them

**Comment by bebatut** [comment]
The page is not finished but we can fill it later and the draft schedule is available for the interview.

**Pull request merged**